README.md

```diff
@@ -58,6 +61,39 @@ the path to the key file, for example:
 If you are deploying to App Engine, Compute Engine, or Container Engine, your
 credentials will be available by default.
 
+#### OAuth 2.0
+
+Many APIs (like Drive, Gmail, and YouTube) require you to use OAuth 2.0 to
+authorize requests on behalf of an authenticated user. For an example using
+the [oauth2-library][oauth2-library], see the [auth sample][auth-sample].
+
+We've also provided a mix task to fetch a token for testing. The following
+command requests a token for the Drive full access scope:
+
+```bash
+$ export GOOGLE_CLIENT_ID=[YOUR-OAUTH-CLIENT-ID]
+$ export GOOGLE_CLIENT_SECRET=[YOUR-OAUTH-CLIENT-SECRET]
+$ mix google_apis.auth https://www.googleapis.com/auth/drive
+Open the following link in your brower:
+https://accounts.google.com/o/oauth2/auth?[some-long-url]
+Enter verification code:
+```
+
+Once you've logged in and authorized the application, copy the code param from
+the web browser's url and paste into the console. The script will then fetch
+your OAuth access token.
+
+```bash
+Token: [your-oauth-token]
+```
+
+You can then use this token for your testing:
+
+```elixir
+connection = GoogleApis.Drive.V3.Connection.new("your-oauth-token")
```

Comment:
This is great! But, what happens when the token expires? Should we include a mix task for refreshing a token?

Reply:
Additionally, it would be great to keep the token in memory and automatically refresh it. And do the same with service accounts. But that's out of scope for this PR.